py-getting-started.Rmd

```diff
@@ -60,29 +60,30 @@ Python has many built-in functions, that allow you to do different things:
 
 ```{python}
 len("Sydney Carton")
-sorted(["Samuel", "Nathaniel", "Augustus", "Tracy"]
+sorted(["Samuel", "Nathaniel", "Augustus", "Tracy"])
 min([4, 2/3, -3, 6.25])
 pow(5,3)
 ```
 
 Part of learning to program is learning some of the core Python functions and 
 what they do; in upcoming chapters, we'll focus on functions that allow you to 
-read in and maniuplate data. At some point it may be helpful to write your 
+read in and manipulate data. At some point it may be helpful to write your 
 *own* functions, which we'll also cover in another chapter. 
 
 ## Running Code via Scripts {#py-getting-started-script}
 
-Can you imagine a sitution where continuing to type code into the console 
+Can you imagine a situation where continuing to type code into the console 
 could become tedious or challenging? 
 
 Here are some examples: 
 
 * Wanting to run the same piece of code many times (you 
 *can* use the up-arrow to see previous commands - try it! - but if you 
 run many commands, your history will quickly become more cluttered. 
-* Needing to re-run a piece of code
-* Most important - when you close Spyder, everything you've typed into the 
```

Comment:
This should not be the case. IPython history persists between sessions by default. I checked with Spyder on my machine and it seems to be working as it should.

Reply:
Ah, I didn't test this, I just assumed. Good to know.